README.md

```diff
@@ -1,29 +1,29 @@
-resin-image-manager
+balena-image-manager
 -------------------
 
-[![npm version](https://badge.fury.io/js/resin-image-manager.svg)](http://badge.fury.io/js/resin-image-manager)
-[![dependencies](https://david-dm.org/resin-io/resin-image-manager.png)](https://david-dm.org/resin-io/resin-image-manager.png)
-[![Build Status](https://travis-ci.org/resin-io/resin-image-manager.svg?branch=master)](https://travis-ci.org/resin-io/resin-image-manager)
-[![Build status](https://ci.appveyor.com/api/projects/status/2nxg1uydksvey0g8?svg=true)](https://ci.appveyor.com/project/jviotti/resin-image-manager)
+[![npm version](https://badge.fury.io/js/balena-image-manager.svg)](http://badge.fury.io/js/balena-image-manager)
+[![dependencies](https://david-dm.org/balena-io/balena-image-manager.png)](https://david-dm.org/balena-io/balena-image-manager.png)
+[![Build Status](https://travis-ci.org/balena-io/balena-image-manager.svg?branch=master)](https://travis-ci.org/balena-io/balena-image-manager)
```

Comment:
Have the svg and png files actually been renamed over there?

Reply:
As in https://github.com/resin-io-modules/resin-sync/pull/157#pullrequestreview-168267862 - these are all autogenerated from github data, so once things get renamed here they'll magically start working.

Reply:
👍